src/InverseSolve/regularizers.jl

```diff
@@ -124,6 +124,50 @@ function wdiffusionReg(m::Vector, mref::Vector, M::AbstractMesh; Iact=1.0, C=[])
    return Rc,dR,d2R
 end # function wdiffusionReg
 
+function wdiffusionAndDepthReg(m::Vector, mref::Vector, M::AbstractMesh; Iact=1.0, C=[])
```

Comment:
It looks like this function combines ```wdiffusionReg``` and ```smallnessReg```, right? If so, we don't need to add a new function for this. In ```pInv```  the regularization function, parameter, and reference model can be stacked into arrays. But I may overlook something here.